powerfulseal/cli/__main__.py

```diff
@@ -131,9 +140,13 @@ def main(argv):
 
     # build cloud inventory
     logger.debug("Fetching the remote nodes")
-    driver = OpenStackDriver(
-        cloud=args.open_stack_cloud,
-    )
+    if args.open_stack_cloud != None:
```

Comment:
Just nitpicking here: I think that `if args.open_stack_cloud is not None` is a more pythonic way - no pressure though 😄